python/ambassador/ir/irtlscontext.py

```diff
@@ -382,29 +382,38 @@ def load_all(cls, ir: 'IR', aconf: Config) -> None:
                     ctx.sourced_by(config)
 
                     ir.save_tls_context(ctx)
-
-                    if ctx.get('hosts'):  # not None and not the empty list
-                        found_termination_context = True
-
-        if ir.edge_stack_allowed and not found_termination_context:
-            # Edge Stack always wants a termination context
-            ctx_name = "fallback-self-signed-context"
-            tls_name = "fallback-self-signed-cert"
-
-            new_ctx = dict(
-                rkey=f"{ctx_name}.99999",
-                name=ctx_name,
-                location="-internal-",
-                hosts=["*"],
-                secret=tls_name,
-                is_fallback=True
-            )
-
-            if not os.environ.get('AMBASSADOR_NO_TLS_REDIRECT', None):
-                new_ctx['redirect_cleartext_from'] = 8080
-
-            ctx = IRTLSContext(ir, aconf, **new_ctx)
-
-            assert ctx.is_active()
-            if ctx.resolve_secret(tls_name):
-                ir.save_tls_context(ctx)
+                    #
+                    # if ctx.get('hosts'):  # not None and not the empty list
+                    #     found_termination_context = True
+
+        # if ir.edge_stack_allowed:
+        #     # We're running Edge Stack. If we have no termination contexts and we
+        #     # have no Hosts...
+        #     host_count = len(ir.get_hosts() or [])
+        #
+        #     ir.logger.info(f"FTC {found_termination_context}, host_count {host_count}")
+        #
+        #     if (host_count == 0) and not found_termination_context:
+        #         ir.logger.info("Creating fallback context")
+        #         # ...then we know that this is an unconfigured installation, so let's set
+        #         # up the fallback TLSContext so we can redirect people to the UI.
+        #         ctx_name = "fallback-self-signed-context"
+        #         tls_name = "fallback-self-signed-cert"
+        #
+        #         new_ctx = dict(
+        #             rkey=f"{ctx_name}.99999",
+        #             name=ctx_name,
+        #             location="-internal-",
+        #             hosts=["*"],
+        #             secret=tls_name,
+        #             is_fallback=True
+        #         )
+        #
+        #         if not os.environ.get('AMBASSADOR_NO_TLS_REDIRECT', None):
+        #             new_ctx['redirect_cleartext_from'] = 8080
+        #
+        #         ctx = IRTLSContext(ir, aconf, **new_ctx)
+        #
+        #         assert ctx.is_active()
+        #         if ctx.resolve_secret(tls_name):
+        #             ir.save_tls_context(ctx)
```

Comment:
Is there a reason to keep this present & commented out?  It looks to me like it's sufficiently similar to the code that appeared in irhost.py to just remove it.

Reply:
None whatsoever, I just missed it in my "clean up the world" pass. Deleting.